DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json

```diff
@@ -95,43 +94,43 @@
                 },
                 {
-                    "title": "3.2 Create a pkcs12 file using the Azure/VM Ip Address",
+                    "title": "2.2 Create a pkcs12 file using the Azure/VM Ip Address",
 		    "description": "Create a pkcs12 certificate using the public IP of the VM instance in Firepower under System->Integration->eStreamer, for more information please see install [guide](https://www.cisco.com/c/en/us/td/docs/security/firepower/670/api/eStreamer_enCore/eStreamereNcoreSentinelOperationsGuide_409.html#_Toc527049443)"
                 },
 		{
-		    "title": "3.3  Test Connectivity between the Azure/VM Client and the FMC",
+		    "title": "2.3  Test Connectivity between the Azure/VM Client and the FMC",
 		    "description": "Copy the pkcs12 file from the FMC to the Azure/VM instance and run the test utility (./encore.sh test) to ensure a connection can be established, for more details please see the setup [guide](https://www.cisco.com/c/en/us/td/docs/security/firepower/670/api/eStreamer_enCore/eStreamereNcoreSentinelOperationsGuide_409.html#_Toc527049430)"
 	        },
                 {
-                    "title": "3.4 Configure encore to stream data to the agent",
+                    "title": "2.4 Configure encore to stream data to the agent",
                     "description": "Configure encore to stream data via TCP to the Microsoft Agent, this should be enabled by default, however, additional ports and streaming protocols can configured depending on your network security posture, it is also possible to save the data to the file system, for more information please see [Configure Encore](https://www.cisco.com/c/en/us/td/docs/security/firepower/670/api/eStreamer_enCore/eStreamereNcoreSentinelOperationsGuide_409.html#_Toc527049433)"
                 }
             ]
         },
+        {
+            "title": "3. Validate connection",
+            "description": "Follow the instructions to validate your connectivity:\n\nOpen Log Analytics to check if the logs are received using the CommonSecurityLog schema.\n\n>It may take about 20 minutes until the connection streams data to your workspace.\n\nIf the logs are not received, run the following connectivity validation script:\n\n> 1. Make sure that you have Python on your machine using the following command: python -version\n\n>2. You must have elevated permissions (sudo) on your machine",
+            "instructions": [
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "WorkspaceId"
+                        ],
+                        "label": "Run the following command to validate your connectivity:",
+                        "value": "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_troubleshoot.py&&sudo python cef_troubleshoot.py  {0}"
```

Comment:
-O cef_troubleshoot.py is missing. Please refer the sample template https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json

Thanks,
Aditi Maheshwari